### DISCUSSION TITLE
Interactive bot

### DISCUSSION THREAD

**Discussion by PrakruthiSomashekar** [deekay2310/calorieApp_server#7] [Ideas]
Hey guys, can we make the application more interactive by adding a chatbot or something similar?

**Comment by deekay2310** [comment]
Sounds good, I plan to take it further and leverage [DialogFlow](https://cloud.google.com/dialogflow/docs) for this purpose. Feel free to provide more inputs!

**Reply by deekay2310** [reply]
@SunidhiHegde has also gone through the documentation and I think we can implement this.
